desc/compute/_basis_vectors.py

```diff
@@ -27,630 +1356,2063 @@ def _e_sub_rho(params, transforms, profiles, data, **kwargs):
     label="\\mathbf{e}_{\\theta}",
     units="m",
     units_long="meters",
-    description="Covariant poloidal basis vector",
+    description="Covariant Poloidal basis vector",
     dim=3,
     params=[],
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["0", "R_t", "Z_t"],
+    data=["R", "R_t", "Z_t", "omega_t"],
 )
 def _e_sub_theta(params, transforms, profiles, data, **kwargs):
-    data["e_theta"] = jnp.array([data["R_t"], data["0"], data["Z_t"]]).T
+    data["e_theta"] = jnp.array(
+        [data["R_t"], data["R"] * data["omega_t"], data["Z_t"]]
+    ).T
     return data
 
 
 @register_compute_fun(
-    name="e_zeta",
-    label="\\mathbf{e}_{\\zeta}",
+    name="e_theta/sqrt(g)",
+    label="\\mathbf{e}_{\\theta} / \\sqrt{g}",
     units="m",
     units_long="meters",
-    description="Covariant toroidal basis vector",
+    description="Covariant Poloidal basis vector divided by 3-D volume Jacobian",
     dim=3,
     params=[],
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["R", "R_z", "Z_z"],
+    data=["e_theta", "sqrt(g)"],
+    axis_limit_data=["e_theta_r", "sqrt(g)_r"],
 )
-def _e_sub_zeta(params, transforms, profiles, data, **kwargs):
-    data["e_zeta"] = jnp.array([data["R_z"], data["R"], data["Z_z"]]).T
+def _e_sub_theta_over_sqrt_g(params, transforms, profiles, data, **kwargs):
+    data["e_theta/sqrt(g)"] = transforms["grid"].replace_at_axis(
```

Comment:
why is this function needed? ah is it due to axis limits?

Reply:
yes